modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1413,8 +1413,8 @@ public String getSchemaType(Schema schema) {
                         // primitive type or model
                         names.add(getAlias(getPrimitiveType(s)));
                     }
-                    return "oneOf<" + String.join(",", names) + ">";
                 }
+                return "oneOf<" + String.join(",", names) + ">";
```

Comment:
@zigen thanks for the fix. I've filed a similar one to fix anyof: https://github.com/OpenAPITools/openapi-generator/pull/1750/files#diff-7cb46fa53f89a458a7b7cb201d2214a8R1399

Ref: https://github.com/OpenAPITools/openapi-generator/pull/1757/files

Reply:
cool!!
thanks for fix.